### PR TITLE
Fixes #2998

### LINKED ISSUE #2998: simple_make missing dnf dep qt5-qtsvg-devel
qt5-qtsvg-devel
ffmpeg-devel

### PR DESCRIPTION
**Pull request by Kengur8**

### CODE REVIEW

File: simple_make.sh [hunk 1]
```diff
@@ -19,7 +19,8 @@ elif which dnf; then
         git qt-devel qt-doc qt-creator qt5-qtsvg opencv-devel \
         openal-soft-devel libXScrnSaver-devel qrencode-devel \
         opus-devel libvpx-devel qt5-qttools-devel glib2-devel \
-        gdk-pixbuf2-devel gtk2-devel libsodium-devel
+        gdk-pixbuf2-devel gtk2-devel libsodium-devel \
+        qt5-qtsvg-devel ffmpeg-devel
```
Comment (zetok) [inline simple_make.sh:23]: https://apps.fedoraproject.org/packages/s/ffmpeg-devel ← there appears to be no `ffmpeg-devel` package for fedora?

**Comment by Kengur8** [comment]
ah, right. I'm using http://negativo17.org/repos/HandBrake

**Comment by sudden6** [comment]
@Kengur8 Can you make this work with vanilla Fedora repositoris?

**Comment by zetok** [comment]
> <linuxmodder> then do  sudo dnf install @"Development Tools"  http://download1.rpmfusion.org/free/fedora/updates/testing/23/x86_64/rpmfusion-free-release-23-0.1.noarch.rpm

In order to be able to install ffmpeg

**Comment by SkyzohKey** [comment]
Here's the RPM for ffmpeg & ffmpeg-devel:

``` bash
rpm -Uvh http://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-stable.noarch.rpm \ http://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-stable.noarch.rpm
```

**Comment by linux-modder** [comment]
@SkyzohKey Firstly, that is  NEVER going to be allowed upstream for packaging as it  breaks the install db  by using rpm -Uvh it would be -ivh  if  even used and  secondly  you have the wrong packages it would NOT be  -stable it would be the default  -release-noarch but seeing as everyone else wishes to make needless bs arguments against my commits for petty shit have fun getting it  fixed I have  resorted to committing locally and  compiling properly. If certain people gave two shits about  function before  sex appeal this project (tox as a whole) would be much better off.

**Comment by zetok** [comment]
Note: required changes done in #3229, would be nice if someone tested that.

**Comment by zetok** [comment]
Closing, since #3229 was merged.

**Pull request closed without merging**